src/org/anjocaido/groupmanager/GroupManager.java

```diff
@@ -98,22 +98,21 @@
  *
  * @author gabrielcouto, ElgarL
  */
+@SuppressWarnings("DeprecatedIsStillUsed")

```

Comment:
We should stop using it ourselves rather than suppress the warning.
The deprecation is there to try and push Vault to stop using it.

Reply:
Vault is what stops it. Looking at [your PR](https://github.com/MilkBowl/Vault/pull/819), they do not want to use `getBukkitPermissions()`
If we suddenly change it, I assume everything will break

Reply:
It will. I set it to deprecated so if they ever change their mind we can simply set the Field to private and its all done.